src/common/src/config.rs

```diff
@@ -676,6 +676,9 @@ pub struct StorageConfig {
     #[serde(default = "default::storage::max_concurrent_compaction_task_number")]
     pub max_concurrent_compaction_task_number: u64,
 
+    #[serde(default = "default::storage::max_upload_task_number")]
+    pub max_upload_task_number: u64,
```

Comment:
The naming will make people thing it is limiting the total number of all uploading tasks but actually it only limits the uploading task for unsealed epoch. Can we make the naming more explicit?

Reply:
ok

Reply:
No. This variable is check the total uploading task but not limit seal data because we worry about it will bring other problem

Reply:
I will limit it in a separate PR